### PR TITLE
change chart into line-chart and create new component for buttons

### PR DESCRIPTION
**Pull request by joanna-by**

### CODE REVIEW

**Comment by mateusziwanczuk** [comment]
@joanna-by Please delete headers from Nav/MainContent.js. They are displaying again in User Panel and Create Pizza sections.

**Comment by joanna-by** [comment]
Fixed

File: src/components/Dashboard/Chart.js [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
+                    fillColor: "rgb(241,205,124)",
+                    strokeColor: "rgb(238,193,90)",
+                    pointColor: "rgb(238,193,90)",
+                    pointStrokeColor: "#fff",
+                    pointHighlightFill: "#fff",
+                    pointHighlightStroke: "rgba(151,187,205,1)",
+                    data: [48, 57, 53, 65, 72, 70]
+                },
+                {
+                    label: "Wykreowana pizze",
```
Comment (mateusziwanczuk) [inline src/components/Dashboard/Chart.js]: Please remember to keep in mind typos in line 23 and 33.

Comment (mateusziwanczuk) [inline src/components/Dashboard/Chart.js]: Please remember about typos ;)

**Comment by mateusziwanczuk** [comment]
Asia, there are still typos :) Line 23 and 33 of Chart.js

**Pull request merged**